fully_documented.txt

```diff
@@ -35,8 +38,12 @@ backpack/extensions/secondorder/diag_hessian/__init__.py
 backpack/extensions/secondorder/diag_hessian/conv1d.py
 backpack/extensions/secondorder/diag_hessian/conv2d.py
 backpack/extensions/secondorder/diag_hessian/conv3d.py
+backpack/extensions/secondorder/sqrt_ggn
```

Comment:
I need general clarification.
- You use just the shortest expression possible
- I use `.py` for files and `.../` for folders
Which one is better? Should we unify our approaches?

Reply:
Unified to your convention.